main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"container/list"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
```

Comment:
Remove the `io/ioutil` import and replace all the occurrences of `ioutil.Discard` to `io.Discard`.

Reply:
Resolved in [9a7af50](https://github.com/oduwsdl/MemGator/pull/148/commits/9a7af50d23e0a22bea98ef936fe5ea38013c5d61)